rpcserver.go

```diff
@@ -178,6 +178,10 @@ var (
 			Entity: "onchain",
 			Action: "read",
 		}},
+		"/lnrpc.Lightning/PendingSweeps": {{
+			Entity: "onchain",
+			Action: "read",
+		}},
```

Comment:
If we decide in the followup to offer bumpfee in a sub server, maybe it should be done with `PendingSweeps` too.

Reply:
Ok, you made that comment yourself in #3140 already

Reply:
Moved to `WalletKit`.